setup.py

```diff
@@ -77,6 +77,7 @@ def find_files(pattern, path=None, root="rez"):
             find_files('*', 'tests/data') +
             find_files('*.exe', 'vendor/distlib'),
         'rezplugins':
+            ['MAIN'] +
```

Comment:
I would much rather not have this. Can we not infer from presence of `rezplugins`?

Reply:
For old-style plugin and to provide a proper fix on issue #960, I think we have to 🤔 
Or instead of as "package_data" file "MAIN", just an empty python script `_main.py` ?

Reply:
reverted.